lib/kino/bridge.ex

```diff
@@ -259,12 +259,12 @@ defmodule Kino.Bridge do
   end
 
   @doc """
-  Requests the child spec for proxy handler with the given function.
+  Requests the child spec for proxy handler with the given plug.
   """
-  @spec get_proxy_handler_child_spec((Plug.Conn.t() -> Plug.Conn.t())) ::
+  @spec get_proxy_handler_child_spec(Kino.Proxy.plug()) ::
           {:ok, {module(), term()}} | request_error()
-  def get_proxy_handler_child_spec(fun) do
-    io_request({:livebook_get_proxy_handler_child_spec, fun})
+  def get_proxy_handler_child_spec(plug) do
```

Comment:
I don't think we need to change here. We can convert the plugs into functions in Kino.Proxy and keep this contract small.

Reply:
Good call, I’ll update soon.